### PR TITLE
[INJICERT-641] add Certify postgres dataprovider plugin

### PR DESCRIPTION
**Pull request by vharsh**

### CODE REVIEW

**Comment by vharsh** [comment]
The above changeset is failing as the postgres plugin isn't merged to `develop` yet.

```
Error: ERROR] Failed to execute goal org.apache.maven.plugins:maven-dependency-plugin:3.5.0:copy (copy) on project artifactory-parent: Unable to find/resolve artifact.: Could not find artifact io.mosip.certify:postgres-dataprovider-plugin:jar:0.3.0-SNAPSHOT in ossrh (https://oss.sonatype.org/content/repositories/snapshots) -> [Help 1]
```

File: artifacts/pom.xml [hunk 1]
```diff
@@ -586,6 +590,15 @@
 						<destFileName>${certify-mock-plugin.fileName}</destFileName>
 						<type>jar</type>
 					</artifactItem>
+<!-- artifactItem section for certify mock ida dataprovider plugin -->
```
Comment (vishwa-vyom) [inline artifacts/pom.xml:593]: comment is confusing, what is the direct relevance to mock ida here ?

File: artifacts/pom.xml [hunk 2]
```diff
@@ -212,13 +212,17 @@
 		<certify-mosip-identity-plugin.fileName>certify-mosip-identity-plugin.jar</certify-mosip-identity-plugin.fileName>
 
 		<!-- certify mock plugin -->
-		<certify-plugin.location>/usr/share/nginx/html/artifactory/libs-release-local/certify/certify-plugin</certify-plugin.location>
```
Comment (vishwa-vyom) [inline artifacts/pom.xml:215]: why is this line removed ?

File: artifacts/pom.xml [hunk 3]
```diff
@@ -212,13 +212,17 @@
...
 		<certify-mock-plugin.version>0.3.0-SNAPSHOT</certify-mock-plugin.version>
 		<certify-mock-plugin.fileName>certify-mock-identity-plugin.jar</certify-mock-plugin.fileName>
 
 		<!-- certify mock ida dataprovider plugin -->
 		<certify-mock-ida-dataprovider-plugin.version>0.3.0-SNAPSHOT</certify-mock-ida-dataprovider-plugin.version>
-		<certify-mock-ida-dataprovider-plugin.fileName>certify-mock-ida-dataprovider-plugin.jar</certify-mock-ida-dataprovider-plugin.fileName>
+        <certify-mock-ida-dataprovider-plugin.fileName>certify-mock-ida-dataprovider-plugin.jar</certify-mock-ida-dataprovider-plugin.fileName>
```
Comment (vishwa-vyom) [inline artifacts/pom.xml:220]: All mocks to be part of the mock-certify-plugin, why do we have a different entry called certify-mock-ida-dataprovider-plugin.jar ?

**Review by vishwa-vyom — CHANGES_REQUESTED** [review]

File: artifacts/pom.xml (hunk 3 above)
Comment (vishwa-vyom) [reply artifacts/pom.xml:221]: @vharsh Is suppose to create a bug for this and attach to this comment, marking it resolved for now.

**Review by vishwa-vyom — APPROVED** [review]

File: artifacts/pom.xml (hunk 3 above)
Comment (vishwa-vyom) [reply artifacts/pom.xml:221]: https://mosip.atlassian.net/browse/INJICERT-643

**Pull request merged**